build.sh

```
@@ -29,6 +29,9 @@ echo "Revision:" $REVISION
 echo "Build Sha:" $BUILD_SHA "vs" $CURRENT_BUILD_SHA "(calculated)"
 echo ""
 
+# Run pre-docker build setup scripts
+(cd ./benchmarks/inference_tract/rust-benchmark && ./setup.sh)
```

`setup.sh` doesn't exist at that directory. Is the benchmark silently failing (since it didn't download a model)?